src/proxy/ClientTable.php

```diff
@@ -107,65 +107,108 @@ public function getOffsetTotal()
     
     public function isComplet()
     {
-        return $this->getRows() == count($this->getContentRows());
+        return $this->getRows() == $this->_contentRowsCount;
     }
     
-    private $_contentRows;
-    
     public function getContentRows()
     {
-        if ($this->_contentRows === null) {
-            $data = [];
-            
-            Console::startProgress(0, $this->getOffsetTotal(), 'Fetch: ' . $this->getName() . ' ');
-            
-            $progress = 1;
-            for ($i=0; $i<$this->getOffsetTotal(); $i++) {
-                $requestData = $this->request($i);
-                
-                if (!$requestData) {
-                    continue;
-                }
-                
-                $data = array_merge($requestData, $data);
-                gc_collect_cycles();
-                Console::updateProgress($progress++, $this->getOffsetTotal());
-            }
-            
-            Console::endProgress();
-            $this->_contentRows = $data;
-        }
-        
-        return $this->_contentRows;
+	    throw new \ErrorException('Not implemented');
     }
-    
-    public function syncData()
-    {
-        Yii::$app->db->createCommand()->truncateTable($this->getName())->execute();
-        
-        $rows = $this->getContentRows();
 
-        $chunks = array_chunk($rows, 25);
-        
-        foreach ($chunks as $match) {
-            Yii::$app->db->createCommand()->batchInsert($this->getName(), $this->cleanUpBatchInsertFields($this->getFields()), $this->cleanUpMatchRow($match))->execute();
-        }
+    private $_contentRowsCount;
+
+    public function getContentRowCount()
```

Comment:
The introduction of this method makes sense to me, but you have to use the method in `luya\admin\proxy\ClientTransfer` on line 35 instead of `count($table->getContentRows())`.